packages/devtools_app/lib/src/shared/object_tree.dart

```diff
@@ -276,18 +276,48 @@ List<DartObjectNode> _createVariablesForAssociations(
 ) {
   final variables = <DartObjectNode>[];
   final associations = instance.associations ?? [];
+
+  // If the key type for the provided associations is not primitive, we want to
+  // allow for users to drill down into the key object's properties. If we're
+  // only dealing with primative types as keys, we can render a flatter
+  // representation.
+  final hasNonPrimitiveKey = associations.fold<bool>(
```

Comment:
should this be `hasPrimitiveKey`?

Reply:
Derp, yup. The coffee hadn't kicked in yet... :-P